IMS/IMS/IMS/DAL/IMSContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
```

Comment:
Add `using IMS.Models;` here. so you can change the following code to 
```C#
        public DbSet<Category> Categories { get; set; }
        public DbSet<Currency> Currencies { get; set; }
        public DbSet<Inventory> Inventories { get; set; }
        public DbSet<ItemPrice> ItemPrices { get; set; }
        public DbSet<Location> Locations { get; set; }
        public DbSet<Price> Prices { get; set; }
        public DbSet<Models.Type> Types { get; set; }
        public DbSet<Item> Items { get; set; }
```

So the code is clean. 

- [ ] Done
- [ ] Approved